### PR TITLE
Vue.js Documentation are Added Under Official Docs

### LINKED ISSUE #1181: [Add]  I want to add vuejs Docs📄
### Site URL

vuejs.org

### Description of the above provided site

- Vuejs is The ProgressiveJavaScript Framework
- An approachable, performant, and versatile framework for building web user interfaces.

### Category

Frontend

### Sub Category

N/a

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Nishitbaria**
Vue.js Documentation are Added in Official Docs in Resources

## Fixes Issue
Closes  #1181 

## Changes proposed

Official Documentation for Vuejs is added under Offical Docs Section

### CODE REVIEW

**Comment by Nishitbaria** [comment]
@rupali-codes  Please review and merge  my pull request

File: database/resources/official-docs.json [hunk 1]
```diff
@@ -40,5 +40,12 @@
     "url": "https://angular.io/docs",
     "category": "resources",
     "subcategory": "officialdocs"
+  },
+  {
+    "name": "Vue.js",
+    "description": "This page contains comprehensive documentation of Vue.js, it's the progressive JavaScript framework.",
```
Comment (rupali-codes) [inline database/resources/official-docs.json:46]:
```suggestion
    "description": "This page contains comprehensive documentation of Vue.js. Vue.js is an progressive JavaScript framework.",
```

**Comment by Nishitbaria** [comment]
Hey @rupali-codes I make the Necessary changes

File: database/resources/official-docs.json [hunk 2]
```diff
@@ -40,5 +40,12 @@
     "url": "https://angular.io/docs",
     "category": "resources",
     "subcategory": "officialdocs"
+  },
+  {
+    "name": "Vue.js",
+    "description": "This page contains comprehensive documentation of Vue.js. Vue.js is an progressive JavaScript framework.",
```
Comment (CBID2) [inline database/resources/official-docs.json:46]: [Proofreading]
```suggestion
    "description": "This page contains comprehensive documentation of Vue.js, a progressive JavaScript framework.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just a small proofreading error

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by k-deepak04 — APPROVED** [review]
LGTM

**Pull request merged**